index.html

```diff
@@ -495,6 +495,462 @@ <h2>
+              <li>|item|["action"] doesn't [=map/exist=] or is not a
+              [=string=].
+              </li>
+              <li>|item|["accept"] doesn't [=map/exist=].
+              </li>
+              <li>|item|["accept"] is not a [=dictionary=].
+              </li>
+              <li>|item|["accept"] [=map/is empty=].
+              </li>
+            </ul>
+          </li>
+          <li>Let |url:URL| be the result of [=URL Parser|parsing=]
+          |item|["action"] with |start_url URL| as the base URL.
+          </li>
+          <li>If |url| is failure, return failure.
+          </li>
+          <li>If |url| is not [=manifest/within scope=] of |scope|, return
+          failure.
+          </li>
+          <li>Let |launch_type:string| be a new [=string=] initialized to
+          "single-client".
+          </li>
+          <li>If |item|["launch_type"] [=map/exists=] and is
+          "multiple-clients", set |launch_type| to |item|["launch_type"].
+          </li>
+          <li>Let |accept:ordered map| be a new [=ordered map=].
+          </li>
+          <li>[=map/for each=] |mime_type_string:string| → |extensions| of
+          |item|["accept"]
+            <ol>
+              <li>If |extensions| is not a [=list=], continue.
+              </li>
+              <li>If |extensions| is [=list/empty=], continue.
+              </li>
+              <li>If |extensions| [=list/contains=] items that are not
+              [=string=]s, continue.
+              </li>
+              <li>If |extensions| [=list/contains=] strings that do not begin
+              with the character `.`, continue.
+              </li>
+              <li>Let |mime_type_parsed:mime type| be the result of running the
+              steps of [=parse a mime type=] on |mime_type_string|.
+              </li>
+              <li>If |mime_type_parsed:mime type| is failure, continue.
+              </li>
+              <li>If |mime_type_parsed/type| is not listed as a top-level type
+              in in [[IANA-MEDIA-TYPES]], continue.
+              </li>
+              <li>Set |accept|[|mime_type_string|] to |extensions|.
+              </li>
+            </ol>
+          </li>
+          <li>If |accept:ordered map| is empty, return failure.
+          </li>
+          <li>Let |file_handler:ordered map| be |ordered map| «[ "action" →
+          |url|, "name" → |item|["name"], "launch_type" → |launch_type|,
+          "launch_type" "accept" → |accept| ]».
+          </li>
+          <li>
+            <a data-cite="appmanifest#dfn-process-image-resources">Process
+            image resources</a> passing |item|["icons"], |file_handler|,
+            |manifest URL|, and "icons".
+          </li>
+          <li>Return |file_handler|.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h2>
+          Execute a file handler launch
+        </h2>
+        <p>
+          The steps to <dfn>execute a file handler launch</dfn> are given by
```

Comment:
@inexorabletash should this wording also be modernized, like in https://github.com/WICG/compute-pressure/issues/96

Reply:
Maybe not? As this isn't actually a web-facing method, i.e. there's no webidl definition for it or its parameters.